pkgs/by-name/wi/wideriver/package.nix

```
@@ -0,0 +1,46 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+  gcc,
+  wayland,
+  wayland-scanner,
+  pkg-config,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wideriver";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "alex-courtis";
+    repo = pname;
```

```suggestion
    repo = "wideriver";
```
See https://github.com/NixOS/nixpkgs/issues/277994